### PR TITLE
docs(tabs): Update tabs topic #2689

### LINKED ISSUE #2689: Update Tabs topic
Update the Tabs topic with the changes introduced with the refactoring IgniteUI/igniteui-angular#8793

### PR DESCRIPTION
**Pull request by astaev**
Closes #2689

### Checklist:
 
 - [ ] check topic's TOC/menu and paragraph headings
 - [ ] link to other topics using `../relative/path.md`
 - [ ] at the References section at the end of the topic add links to topics, samples, etc
 - [ ] reference API documentation instead of adding a section with API

<br />

 - [ ] use valid component names - [Data] Grid, `IgxSelectComponent`, `<igx-combo>`
 - [ ] use spell checker tool (VS Code, Grammarly, Microsoft Editor)
 - [ ] add inline `code blocks` for the names of classes / tags / properties
 - [ ] add language descriptor for the ```code blocks```
 - [ ] check broken links (use browser add-on)
 - [ ] check if sample is working and fully visible in the topic
 - [ ] check if sample is working and fully visible in the StackBlitz
 - [ ] check if code blocks match the code in StackBlitz demo

<br />

 - [ ] follow SEO guidelines to fill topic's metadata ([SEO guidelines](https://infragisticsinc297.sharepoint.com/:w:/g/Groups/marketing/EWz9InT4FDlErHCumxsKGY4Bd8H03yhRWxDFk47luRz-_Q?e=S5wWcx))

<br />

 - [ ] do not resolve requested changes (leave that to the reviewer)
 - [ ] add `pending-localization` label when the review of the PR is done
 - [ ] add a member from the localization team to translate it

### CODE REVIEW

File: en/components/tabs.md [hunk 1]
```diff
@@ -38,15 +38,50 @@ import { IgxTabsModule } from 'igniteui-angular';
 export class AppModule {}
 ```
 
-We set the names of tabs using the [`label`]({environment:angularApiUrl}/classes/igxtabsgroupcomponent.html#label) attribute. Any content provided between the opening and closing `igx-tabs-group` tags will appear in the respective tab group as content.
+We set the tabs header by providing a content between [`igx-tab-header`]({environment:angularApiUrl}/classes/igxtabheader.html}). To set the tabs name we simply add a span with [`igxTabHeaderLabel`]({environment:angularApiUrl}/classes/igxtabheaderlabeldirective.html) directive, between the `igx-tab-header` tags. Any content that will appear as a tab item's content should be added between [`igx-tab-content`]({environment:angularApiUrl}/classes/igxtabcontent.html}) tags.
```
Comment (gedinakova) [inline en/components/tabs.md:41]: '...by providing content between [`igx-tab-header`]({environment:angularApiUrl}/classes/igxtabheader.html}) tags. To set the tab's name we simply add a span with [`igxTabHeaderLabel`]({environment:angularApiUrl}/classes/igxtabheaderlabeldirective.html) directive.

File: en/components/tabs.md [hunk 2]
```diff
@@ -61,41 +96,30 @@ If the sample is configured properly, the final result should look like that:
...
-  </igx-tabs>
-
-  <igx-tabs tabsType="fixed">
-    <igx-tabs-group label="Home">Home content.</igx-tabs-group>
-    <igx-tabs-group label="Recent Contacts">Recent contacts list.</igx-tabs-group>
-    <igx-tabs-group label="More">More detailed contact information.</igx-tabs-group>
-  </igx-tabs>
-```
 
+Sample below demostrates how tabs get aligned when switching between tabAlignment property values.
```
Comment (gedinakova) [inline en/components/tabs.md:107]: `tabAlignment`

File: en/components/tabs.md [hunk 3]
```diff
@@ -61,41 +96,30 @@ If the sample is configured properly, the final result should look like that:
...
+           iframe-src="{environment:demosBaseUrl}/layouts/tabs-alignment" >
 </code-view>
 
 
 <div class="divider"></div>
 
 ## Customizing Tabs
 
-Let's modify the tabs and make them more appealing by including icons using the [`icon`]({environment:angularApiUrl}/classes/igxtabsgroupcomponent.html#icon) input. The `igx-tabs` component is compatible with the Material Design
+Let's modify the tabs and make them more appealing by including icons using the [`igxTabHeaderIcon`]({environment:angularApiUrl}/classes/igxtabsgroupcomponent.html#icon) directive. The `igx-tabs` component is compatible with the Material Design
```
Comment (gedinakova) [inline en/components/tabs.md:119]: The link is the old one.

File: en/components/tabs.md [hunk 4]
```diff
@@ -61,41 +96,30 @@ If the sample is configured properly, the final result should look like that:
...
 [**Icons**](https://material.io/icons/) so it will be very easy to adopt them in your application.
 
-First add the Material Icons in your 'styles.scss' file in the main application folder.
+First add the Material Icons in your 'styles.scss' file in the main application folder. Next add `igx-icon` with [`igxTabHeaderIcon`]({environment:angularApiUrl}/classes/igxtabheadericondirective.html}) directive set, as a child of [`igx-tab-header`]({environment:angularApiUrl}/classes/igxtabheader.html}) .
```
Comment (gedinakova) [inline en/components/tabs.md:122]: First, ... Next, ....

**Review by gedinakova — CHANGES_REQUESTED** [review]

File: en/components/tabs.md (hunk 1 above)
Comment (astaev) [reply en/components/tabs.md]: Fixed

File: en/components/tabs.md (hunk 3 above)
Comment (astaev) [reply en/components/tabs.md]: Fixed

File: en/components/tabs.md (hunk 4 above)
Comment (astaev) [reply en/components/tabs.md]: Fixed

File: en/components/tabs.md (hunk 2 above)
Comment (gedinakova) [reply en/components/tabs.md:107]: Put quotes around tabAlignment

**Review by gedinakova — CHANGES_REQUESTED** [review]

File: en/components/tabs.md (hunk 1 above)
Comment (gedinakova) [reply en/components/tabs.md]: The first sentence "We set the tabs header by **providing a content between [`igx-tab-header`]**" -> "**providing content to [`igx-tab-header`]**" sounds better.

File: en/components/tabs.md (hunk 2 above)
Comment (astaev) [reply en/components/tabs.md]: Done

File: en/components/tabs.md (hunk 1 above)
Comment (astaev) [reply en/components/tabs.md]: Changed

**Review by gedinakova — APPROVED** [review]

**Review by gedinakova — APPROVED** [review]

**Pull request merged**